kolibri_instant_schools_plugin/tasks.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import os
+import requests
+
+from kolibri.core.tasks.decorators import register_task
+from kolibri.core.tasks.job import State
+from kolibri.core.tasks.main import job_storage
+
+USER_DATA_POST_FUNC = "kolibri_instant_schools_plugin.auth.api.send_user_data_to_opco"
+SCH_REQUEUE_FAILED_USER_JOB_ID = "100"
+SCH_ANONYMIZE_JOB_ID = "101"
```

Comment:
@jredrejo should we make the enum values strings of the same name as their token? Just worried someone else might decide to make an ad hoc job with the same ID and throw things out of whack -- or is there some mechanism to avoid conflicts like that already in place?

Reply:
I'm not sure if anything other than `int` and `uuid` can be used for the id. If a `string` can be used, I think your idea is a good solution. 
As far as I know there's not any mechanism to avoid conflicts. The only "known" ids are 1 and 2 for the telemetry and vacuum jobs. Others usually get an uuid automatically assigned. I picked those numbers because I thought they were big enough to avoid conflicts after 1 & 2, but it's quite clear than using strings would be much safer if possible.